### DISCUSSION TITLE
Maybe don't include bots in the feed?

### DISCUSSION THREAD

**Discussion by dhirschfeld** [community/community#13408] [Other Feature Feedback, Questions, & Ideas] [answered]
The new feed definitely isn't for me. I can't see myself ever wanting to use it regardless, however, in case it appeals to some people, it could be vastly improved by allowing you to choose to not see updates from certain people... or bots forking literally thousands of repos.

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
